app/assets/stylesheets/customsdecexfrontend-app.scss

```diff
@@ -49,6 +51,6 @@
 $govuk-assets-path: '/customs-exports-internal/assets/';
 $govuk-images-path: "/customs-exports-internal/assets/lib/govuk-frontend/govuk/assets/images/";
 $govuk-fonts-path: "/customs-exports-internal/assets/lib/govuk-frontend/govuk/assets/fonts/";
-
 @import 'lib/govuk-frontend/govuk/all';
 
+@import 'partials/_step-by-step-navigation';
```

Comment:
`@import 'lib/govuk-frontend/govuk/all';` should be the last import. 
Please change and test if the layouts look good

Reply:
I compared this code to the one existing in the movements-frontend. I believe the reason that forced Maciej (while he was working on a similar issue on movements-frontend) to move the `step-by-step-navigation` beyond `govuk-frontend` and to the edge of the CSS chain is to preserve compatibility of the legacy pages with the new GDS model. The trick is done using an override at the end, here (last line):

````
    @import '_step-by-step-navigation-overrides.scss';
````

Reply:
The reason why `@import 'partials/_step-by-step-navigation';` is after `@import 'lib/govuk-frontend/govuk/all';` is because it requires some scss variables or functions (if they are called this way), so they need to be imported first. 
Maybe it could work if `partials/_step-by-step-navigation` had proper imports inside it, but did not come into my mind until now.